### PR TITLE
Remove the getAlertRule method

### PR DESCRIPTION
**Pull request by johntdyer**
Did this to align up w/ the alerts functions, especially since the api always returns an array.  This lines up better w/ established methods...

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,4 +2,7 @@ module github.com/william20111/go-thousandeyes
 
 go 1.13
 
-require github.com/stretchr/testify v1.4.0
+require (
+	github.com/davecgh/go-spew v1.1.1
```
Comment (william20111) [inline go.mod:6]: do we need this lib?

File: go.sum [hunk 2]
```diff
@@ -1,5 +1,7 @@
 github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
+github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```
Comment (william20111) [inline go.sum:4]: same as above

File: go.mod (hunk 1 above)
Comment (johntdyer) [reply go.mod:6]: shit, no, that was me testing... my bad

Comment (johntdyer) [reply go.mod]: fixed

**Pull request merged**